include/vctr/Expressions/Core/Multiply.h

```diff
@@ -78,6 +78,19 @@ class MultiplyVectors : ExpressionTemplateBase
         return dst;
     }
 
+
+    VCTR_FORCEDINLINE void evalVectorOpMultiplyAccumulate (value_type* dst) const
+    requires is::suitableForAccelerateRealFloatBinaryVectorOp<SrcAType, SrcBType, value_type, detail::dontPreferIfIppAndAccelerateAreAvailable> && is::anyVctr<SrcAType> && is::anyVctr<SrcBType>
+    {
+        Expression::Accelerate::multiplyAdd (srcA.data(), srcB.data(), dst, dst, size());
+    }
+
+    VCTR_FORCEDINLINE void evalVectorOpMultiplyAccumulate (value_type* dst) const
+    requires is::suitableForIppRealOrComplexComplexFloatBinaryVectorOp<SrcAType, SrcBType, value_type, detail::preferIfIppAndAccelerateAreAvailable> && is::anyVctr<SrcAType> && is::anyVctr<SrcBType>
```

Comment:
`suitableForIppRealOrComplexComplexFloatBinaryVectorOp` Can we come up with a longer one? 😅

Reply:
Hahaha, you can't imagine how long I thought about the shortest possible names that communicate what these concepts do. 🤯

Reply:
Nope that's intentional. It converts a complex source into a complex destination. For e.g. abs this would be `RealOrComplexReal`. As I said I tried to keep a balance between verbosity and length. But I'm open for renaming proposals as long as they result in a consistent pattern across all those concepts